### PR TITLE
33 a little metadata

### PR DESCRIPTION
**Pull request by absurdfarce**

### CODE REVIEW

**Comment by absurdfarce** [comment]
Apologies, this one got a little confusing, mainly because I am an idiot.

This started life as a PR off of the work being done for https://github.com/datastax/adelphi/issues/40 (specifically https://github.com/datastax/adelphi/pull/51).  When that issue was closed I made the mistake of deleting the feature branch _before_ rebasing the other PR off of master; that in turn effectively killed the first PR.  The simplest way to get back to a good state seemed to be to pull in the branch for the old PR itself ("git pull origin pull/55/head") and handle any last minute conflicts that may have come up.  The result of that process is what you see in this PR.

The "extra" commits referenced above presumably come from the fact that HEAD of pull/55 would have included the Github work (since I originally branched off of that).  Since those commits are now in master they should be silently ignored here.

The contents of this PR should match up to what's in the prior version (https://github.com/datastax/adelphi/pull/55) with a few changes based on last-minute changes in the issue 40 PR.

File: python/adelphi/bin/adelphi [hunk 1]
```diff
@@ -29,25 +34,29 @@ import click
...
 @click.option('--password', help='Database password')
 @click.option('--keyspaces',
               help='Comma-separated list of keyspaces to include. If not specified all non-system keypaces will be included')
 @click.option('--rf', type=int, help='Replication factor to override original setting. Optional.')
-@click.option('--anonymize/--no-anonymize', default=True, help="Enable/disable schema anonymization")
-@click.option('--output-dir', help='Schema files will be written to this directory. If not specified schemas will be written to stdout.')
+@click.option('--anonymize/--no-anonymize', default=True, show_default=True, help="Enable/disable schema anonymization")
+@click.option('--output-dir', help='Directory schema files should be written to. If not specified, it will write to stdout')
+@click.option('--purpose', help='Comments on the anticipated purpose of this schema.  Optional')
+@click.option('--maturity', help='Comments on the maturity of this schema.  Optional')
```
Comment (grighetto) [inline python/adelphi/bin/adelphi:57]: I'm wondering if we should offer some guidance here, like list some options in help: alpha, beta, prod, staging, ... I think it gives us a better chance of people actually filing this in and having a consistent format.
I'm not saying we should enforce any of these values, but simply propose them.
@absurdfarce wdyt?

**Review by grighetto — APPROVED** [review]
Pending question around maturity command help.
Otherwise +1

File: python/adelphi/bin/adelphi (hunk 1 above)
Comment (absurdfarce) [reply python/adelphi/bin/adelphi:57]: Per external conversation we're going to be doing exactly this as a middle ground between a free text field vs. an enum of allowed values.  Change to follow shortly.

**Pull request merged**